Modules/_ctypes/_ctypes_test.c

```diff
@@ -13,6 +13,10 @@
 
 #include <Python.h>
 
+#ifdef __STDC_IEC_559_COMPLEX__
```

Comment:
Since you're using quite a lot this directive, maybe we could just make it something like `HAVE_IEC_60559_COMPLEX` which would check also for the presence of `__STDC_NO_COMPLEX__`.

Reply:
Marking this comment as obsolete since we shouldn't have both macros defined at the same time.